.github/workflows/main.yml

```diff
@@ -66,6 +66,12 @@ jobs:
       - name: 'concatenate runtime.js, main.js, and polyfills.js into lf-documentation.js'
         run: (cat ./dist/lf-documentation/runtime.js; echo ""; cat ./dist/lf-documentation/polyfills.js; echo ""; cat ./dist/lf-documentation/main.js; echo) > ./dist/lf-documentation/lf-documentation.js
 
+      - name: 'gulp replacePlaceholdersInDocumentation'
```

Comment:
this needs to be after replacing the scripts for the stylesheets because those contain CDN_VERSION?